detectors/sPHENIX/Fun4All_G4_sPHENIX.C

```diff
@@ -466,6 +473,17 @@ int Fun4All_G4_sPHENIX(
 
   if (Enable::USER) UserAnalysisInit();
 
+  //======================
+  // Run KFParticle on evt
+  //======================
+  Enable::KFPARTICLE = false;
```

Comment:
Shall we put the ` Enable::*` section to top of the macro, after `Enable::PARTICLEFLOW`?

Reply:
Also since `Enable::KFPARTICLE` was default to `false`, we could change this line to 
```
// Enable::KFPARTICLE = true;
```
to be consistent with others